tests/js/arrays/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -165,6 +165,416 @@ printWidth: 80
 ================================================================================
 `;
 
+exports[`numbers-in-args.js format 1`] = `
+====================================options=====================================
+parsers: ["babel", "flow", "typescript"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+expect(bifornCringerMoshedPerplexSawder.getArrayOfNumbers()).toEqual(
+  [1, 2, 3, 4, 5]
+);
+
+expect(bifornCringerMoshedPerplexSawder.getLongArrayOfNumbers()).toEqual(
+  [
+		66,57,45,47,33,53,82,81,76,78,10,78,15,98,24,29,32,27,28,76,41,65,84,35,
+		97,90,75,24,88,45,23,75,63,86,24,39,9,51,33,40,58,17,49,86,63,59,97,91,
+		98,99,5,69,51,44,34,69,17,91,27,83,26,34,93,29,66,88,49,33,49,73,9,81,4,
+		36,5,14,43,31,86,27,39,75,98,99,55,19,39,21,85,86,46,82,11,44,48,77,35,
+		48,78,97
+	]
+);
+
+=====================================output=====================================
+expect(bifornCringerMoshedPerplexSawder.getArrayOfNumbers()).toEqual([
+  1, 2, 3, 4, 5,
+]);
```

Comment:
Same problem as https://github.com/prettier/prettier/issues/9613#issuecomment-762544486

Let's keep it this way for now.